atlp-rwanda/e-commerce-lydia-32-fn
# view user profile

### What does this PR do?

- user should update profile
- user should change password with old password

### Description of Task to be completed?
Users should be able to amend/edit their profiles depending on a change of preference to the entire disclosure of the individual user

### How should this be manually tested?
1. login 
2. click your name appear on top right corner
3. click profile
4. update your profile details


### Any background context you want to provide?

#### N/A

### Screenshots (if appropriate)

#### N/A

### Questions:

#### N/A

### What are the relevant pivotal tracker stories

(https://www.pivotaltracker.com/story/show/187570825)
